tasks/sep/zora-002-fp-upgrade/VALIDATION.md

```diff
@@ -0,0 +1,52 @@
+# Validation
+
+This document can be used to validate the state diff resulting from the execution of the upgrade
+transaction. For each contract listed in the state diff, please verify that no contracts or state
+changes shown in the Tenderly diff are missing from this document.
+
+Additionally, please verify that for each contract:
+
+- The following state changes (and none others) are made to that contract. This validates that no
+  unexpected state changes occur.
+- All addresses (in section headers and storage values) match the provided name, using the
+  Etherscan and Superchain Registry links provided. This validates the bytecode deployed at the
+  addresses contains the correct logic.
+- All key values match the semantic meaning provided, which can be validated using the storage
+  layout links provided.
+
+## State Overrides
+
+### SystemOwnerSafe (0x1Eb2fFc903729a0F03966B917003800b145F56E2)
+
+- Key: 0x0000000000000000000000000000000000000000000000000000000000000004
+  - Value: 0x0000000000000000000000000000000000000000000000000000000000000001
+  - Description: Enables the simulation by setting signing threshold to 1
+
+## State Changes
+
+### SystemConfigProxy (0xB54c7BFC223058773CF9b739cC5bd4095184Fb08)
+
+- Key: 0x52322a25d9f59ea17656545543306b7aef62bc0cc53a0e65ccfa0c75b97aa906
+  - Value: 0x000000000000000000000000A983A71253Eb74e5E86A4E4eD9F37113FC25f2BF
+  - Description: Slot at keccak(systemconfig.disputegamefactory)-1 set to address of DisputeGameFactoryProxy deployed via upgrade script
```

Comment:
For all of these can you follow the standard format and include the before value? This way it's more familiar for signers. Similarly see https://github.com/ethereum-optimism/superchain-ops/blob/5268dba826a175f0af93b17d209a7b2fa6a0baf6/tasks/sep/027-holocene-system-config-upgrade-and-init-multi-chain/VALIDATION.md for a recent example. So for example this would basically become:

```
* **Key**: `0x52322a25d9f59ea17656545543306b7aef62bc0cc53a0e65ccfa0c75b97aa906`
* **Before**: `0x0000000000000000000000000000000000000000000000000000000000000000`
* **After**: `0x000000000000000000000000a983a71253eb74e5e86a4e4ed9f37113fc25f2bf`
* **Meaning**: Slot at `keccak(systemconfig.disputegamefactory)-1` stores the address of DisputeGameFactoryProxy deployed via upgrade script.
```